bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingHandlerCallback.java

```diff
@@ -127,6 +131,17 @@
      */
     ChannelBuilder editChannel(Thing thing, ChannelUID channelUID);
 
+    /**
+     * Creates a list of {@link ChannelBuilder}s which are preconfigured with values from the given channel group type.
+     *
+     * @param channelGroupUID the UID of the channel group to be created
+     * @param channelGroupTypeUID the channel group type UID for which the channel should be created
+     * @return a list of preconfigured ChannelBuilders
+     * @throw {@link IllegalArgumentException} if the referenced channel group type is not known
+     */
+    List<ChannelBuilder> createChannelGroupBuilder(ChannelGroupUID channelGroupUID,
```

Comment:
I wonder if this is the best method name - it suggests that it creates a "ChannelGroupBuilder", but such a class does not exist. Instead, it creates a list of ChannelBuilders. So shouldn't it be hence called `createChannelBuilders` instead?